tensorflow/python/keras/losses.py

```diff
@@ -1843,15 +1843,34 @@ def deserialize(name, custom_objects=None):
 
 @keras_export('keras.losses.get')
 def get(identifier):
-  """Retrieves a Keras loss function.
+  """Retrieves a Keras loss as a `function`/`Loss` class instance.
+
+  The `identifier` may be the string name of a loss function or `Loss` class.
+
+  >>> loss = tf.keras.losses.get("categorical_crossentropy")
+  >>> type(loss)
+  <class 'function'>
+  >>> loss = tf.keras.losses.get("CategoricalCrossentropy")
+  >>> type(loss)
+  <class 'tensorflow.python.keras.losses.CategoricalCrossentropy'>
```

Comment:
Inside Google the class is different (has a prefix). Let's replace with `<class '...tensorflow.python.keras.losses.CategoricalCrossentropy'>`

(hopefully this works, if it doesn't we'll have to rewrite the test somehow